code/game/machinery/OpTable.dm

```diff
@@ -53,6 +56,12 @@
 		user_unbuckle_mob(user)
 	return
 
+/obj/machinery/optable/attackby(obj/item/I, mob/user, var/params)
```

Comment:
This isn't a great design I think

First thing that comes to mind, tools. Before allowing things to be placed on the altar, you need to allow for tools to deconstruct it.

The second issue, unlike a normal table where everything is the surface, here the surface of an optable or altar is only a part of the sprite, your code will allow people to place items in wierd looking locations

Thirdly, its often going to have a patient on it, in which case you'll only be able to click the wierd side locations.

The solution I was planning on is to implement a whitelist of placeable items, with a corresponding pixel offset. And populate that whitelist with just one thing for the altar - the cruciform. I'd have it slot into a specific position somewhere on the side, a premade niche

Reply:
Whitelist is kinda restrictive. Why do we need it, when we can so same thing tables do?

Reply:
well issue 2 mentioned above. But besides that? 

These are really meant for putting people on, not objects. The only reason you need to put anything on it is for a ritual, and thats just one thing

I guess using an altar as a table also makes sense, but only when there isn't a person on it. And you'd really need to restrict the clickable range so things can only be placed on the actual surface of it